modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java

```diff
@@ -659,6 +659,14 @@ public List<CommerceOrder> getCommerceOrders(
 			groupId, commercePaymentMethodKey);
 	}
 
+	@Override
+	public List<CommerceOrder> getCommerceOrdersByAccount(
```

Comment:
I really like the idea of the ModelListener for AccountEntry name changes!

But this method, isn't it pretty much the same as

public List<CommerceOrder> getCommerceOrdersByCommerceAccountId(
		long commerceAccountId, int start, int end,
		OrderByComparator<CommerceOrder> orderByComparator)

that you can find at line 671(679 with your changes)?